modules/ROOT/pages/advanced/usage-with-module-loaders/reference/localization.adoc

```diff
@@ -5,15 +5,17 @@
 
 == Overview
 
-* <<usingcommunitylocalizations,Using community localizations>>
-* <<usingpremiumself-hostedlocalizations,Using premium self-hosted localizations>>
+* xref:usingcommunitylocalizations[Using community localizations]
+* xref:usingpremiumself-hostedlocalizations[Using premium self-hosted localizations]
 
+[[usingcommunitylocalizations]]
 == Using community localizations
 
 :forModuleLoaders: true
 include::partial$misc/using-community-lang-packs.adoc[]
 :forModuleLoaders: false
 
+[[usingpremiumself-hostedlocalizations]]
 == Using premium self-hosted localizations
 
 :editorcomponent: localization
```

Comment:
The line below here has a conversion issue as the include is using an incorrect heading level. It should be something like this:
```
include::partial$module-loading/bundling-ref-example.adoc[leveloffset=+1]
```

Reply:
I'm changing it and I also changed: 
`language: 'sv_SE',`
to
`language: 'sv_SE'`,
I just moved the `,` out of the block

Reply:
done